panel/src/components/Forms/Blocks/Types/Text.vue

```diff
@@ -1,10 +1,8 @@
 <template>
-	<k-writer
+	<component
+		:is="component"
 		ref="input"
-		:inline="textField.inline"
-		:marks="textField.marks"
-		:nodes="textField.nodes"
-		:placeholder="textField.placeholder"
+		v-bind="textField || $props"
```

Comment:
What's the fallback to `$props` for? Isn't the only prop `endpoints` and wouldn't fit?

Reply:
Yes. `textField` can be empty as `{}`. Thus fallback to default props.

Reply:
But `$props` is just `{ endpoint : ...}` for `Blocks/Types/Text.vue`. Passing that to the component doesn't make sense, does it?

Reply:
Yes I know. But if it doesn't make sense to you, we can remove it. I was afraid of breaking things without `endpoints`.

Reply:
But we weren't passing `endpoints` before this PR - do you think it would break?

Reply:
I found the source of `endpoints` implementation: https://github.com/getkirby/kirby/commit/81d489d19e5deeab406867bb47eb55cdcd1d112b . I think we doesn't need `endpoints` anymore.